src/server/game/AI/CoreAI/UnitAI.h

```diff
@@ -198,14 +198,14 @@ class UnitAI
     virtual ~UnitAI() {}
 
     virtual bool CanAIAttack(Unit const* /*target*/) const { return true; }
-    virtual void AttackStart(Unit* /*target*/);
+    virtual void AttackStart(Unit* /*target*/);    /// @biref Use to start attacking a target. Called just before JustEngagedWith()
```

Comment:
brief?

Reply:
Doxygen usage (https://www.cs.cmu.edu/~410/doc/doxygen.html). @Brief give a short description of the method. @param describe parameters... and there are more keyword that we can use (@note, @return....). It's also use by some IDE (Visual Studio by example).